lib/parallel_tests/cli.rb

```diff
@@ -207,6 +225,17 @@ def parse_options!(argv)
           "Use 'isolate'  singles with number of processes, default: 1."
         ) { |n| options[:isolate_count] = n }
 
+        opts.on(
+          "--retain-exit-status",
```

Comment:
```suggestion
          "--highest-exit-status",
```

Reply:
Ah, yeah, this is a better option name. Updated in 3a62cb0